web_src/js/features/repo-issue.js

```diff
@@ -288,14 +290,15 @@ export function initRepoPullRequestAllowMaintainerEdit() {
       $.ajax({url, type: 'POST',
         data: {_csrf: csrfToken, allow_maintainer_edit: checked},
         error: () => {
-          $checkbox.popup({
-            content: promptError,
-            onHidden: () => {
-              // the error popup should be shown only once, then we restore the popup to the default message
-              $checkbox.popup({content: promptTip});
+          tippy.hide();
+          tippy.setContent(promptError);
+          tippy.show();
+          tippy.setProps({
+            onHide: (tippy) => {
+              tippy.setContent(promptTip);
+              tippy.setProps({onhide: undefined});
```

Comment:
onHide or onhide?

Reply:
`onHide`. It's meant to remove the event listener, but it's probably not the intended way, will look into how to do it properly.

Reply:
Don't think tippy offers any other way to remove such listeners, but it appears to work with setting to `undefined` at least, e.g. it does not add multiple listeners.